### DISCUSSION TITLE
How do I run arch.appx in windows terminal

### DISCUSSION THREAD

**Discussion by joaodejesus** [yuk7/ArchWSL#275] [Q&A]
Good Morning! I installed Arch through .appx and when running the windows terminal I noticed that the arch profile does not appear. I would like to know where this arch installation is stored and how I can run it through the windows terminal.

**Comment by joaopasantos** [comment]
I think that installing through the App Installer doesn't create the profile automatically. But you can do it yourself, go into Settings>Add a new profile and put "wsl.exe -d Arch" in the "Command line" field. That way you can customize everything else too.
As for where the Arch installation is, this might give you a clue: [Where are Windows store apps installed and where do they save data](https://www.techmesto.com/windows-store-apps-installed-save-data/).
